system-variables.md

```diff
@@ -1325,6 +1325,17 @@ explain select * from t where age=5;
 SET tidb_query_log_max_len = 20;
 ```
 
+### `tidb_rc_read_check_ts` <span class="version-mark">从 v6.0.0 版本开始引入</span>
+
+- 作用域：SESSION | GLOBAL
+- 默认值：`0`
+- 悲观事务中当 `READ-COMMITTED` 隔离级别启用之后，每个事务中 `SELECT` 语句会尝试获取一个新的 `timestamp` 用于数据读取。当开启此变量之后，`SELECT` 语句会尝试使用前一个有效的 `timestamp` 进行数据读取，初始值为事务的 `start_ts`。如果整个读取过程没有遇到更新版本，则返回结果给客户端且 `SELECT` 语句执行成功。当读到过程遇到更新的版本时，如果当前 `TiDB` 尚未向客户端回复数据，则尝试重新获取一个新的 `timestamp` 重试此语句。如果已经回复过客户端数据，则 `TiDB` 会向客户端报错。回复客户端数据量受 `tidb_init_chunk_size` 和 `tidb_max_chunk_size` 控制。在使用 `READ-COMMITTED` 隔离级别且单个事务中 `SELECT` 语句较多，读写冲突较少的场景，可通过开启此变量用以避免获取全局 `timestamp` 带来的延迟和开销。
+
```

Comment:
I suggest you only mention that it's an optimization for RC transactions that's suitable for read-heavy workloads, and put these implementation details in a supplementary information block (so that users don't have to understand it). Like this:

```markdown
> **说明：**
>
> * 在这里解释取 ts 之类的内部原理
```

I think a user may not understand what it means by *retrieving a timestamp* or *finding a newer version*. It's too complex for user.